packages/flutter_tools/lib/src/base/logger.dart

```diff
@@ -1145,27 +1145,28 @@ typedef SlowWarningCallback = String Function();
 /// a [Status] or one of its subclasses.
 abstract class Status {
   Status({
-    this.onFinish,
+    VoidCallback? onFinish,
     required Stopwatch stopwatch,
-    this.timeout,
-  }) : _stopwatch = stopwatch;
+    Duration? timeout,
+  })  : _timeout = timeout,
+        _onFinish = onFinish,
+        _stopwatch = stopwatch;
 
-  final VoidCallback? onFinish;
-  final Duration? timeout;
+  final VoidCallback? _onFinish;
+  final Duration? _timeout;
 
   @protected
   final Stopwatch _stopwatch;
 
   @protected
-  String get elapsedTime {
+  String get _elapsedTime {
```

Comment:
Does this still need to be marked as `@protected` (here and elsewhere where we have the annotation on library private members)?

Reply:
Thanks for catching this! Fixed in [make ticks readonly, remove unnecessary protected annotations](https://github.com/flutter/flutter/pull/158492/commits/d81d598225749f37da3db436a6c6e34de00d430d)